tidb-cloud/vector-search-data-types.md

```diff
@@ -77,6 +77,80 @@ INSERT INTO vector_table VALUES (2, '[0.3, 0.5]');       -- 2 dimensions vector,
 
 However you cannot build a [Vector Search Index](/tidb-cloud/vector-search-index.md) for this column, as vector distances can be only calculated between vectors with the same dimensions.
 
+## Comparison
+
+Vector data types can be compared using the [compare operators](/functions-and-operators/operators.md) you are already familiar with, like `=`, `!=`, `<`, `>`, `<=`, `>=`. For a full list of compare operators and functions available for Vector data types, see [Vector Functions and Operators](/tidb-cloud/vector-search-functions-and-operators.md).
+
+Vector data types are compared element-wise numerically. Examples:
+
+- `[1] < [12]`
+- `[1,2,3] < [1,2,5]`
+- `[1,2,3] = [1,2,3]`
+- `[2,2,3] > [1,2,3]`
+
+Vectors with different dimensions are compared using Lexicographical comparison, with the following properties:
+
+- Two Vectors are compared element by element, each element is compared numerically.
+- The first mismatching element defines which sequence is lexicographically _less_ or _greater_ than the other.
+- If one sequence is a prefix of another, the shorter sequence is lexicographically _less_ than the other.
+- If two sequences have equivalent elements and are of the same length, then the sequences are lexicographically _equal_.
+- An empty sequence is lexicographically _less_ than any non-empty sequence.
+- Two empty sequences are lexicographically _equal_.
+
+Examples:
+
+- `[] < [1]`
+- `[1,2,3] < [1,2,3,0]`
+
+Note that you may want [explicit cast](#cast) from string to vector when comparing vector constants, otherwise the comparison will be done by string values:
```

Comment:
```suggestion
When comparing vector constants, consider performing an [explicit cast](#cast) from string to vector to avoid comparisons based on string values:
```